core/templates/dev/head/pages/landing/LandingPage.js

```diff
@@ -0,0 +1,167 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for landing page.
+ */
+
+oppia.constant('LANDING_PAGE_DATA', {
+  maths: {
+    fractions: {
+      collection_id: '4UgTQUc1tala',
+      student: {
+        row_1: {
+          heading: 'Fractions just got easier.',
+          image: 'hero_bakery_combined.png'
```

Comment:
@seanlip, We can make a pattern for naming image/video file (Like Fraction_row_1.png) and can clean this us but I'm not sure whether that will be a good idea?

Reply:
I think OK to leave as-is for now.